modules/metastore/src/ResourceMapper.php

```diff
@@ -32,17 +33,30 @@ class ResourceMapper {
   private $store;
 
   /**
-   * Event dispatcher service.
+   * Entity type manager service.
    *
-   * @var \Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
    */
-  private $eventDispatcher;
+  private EntityTypeManagerInterface $entityTypeManager;
+
+  /**
+   * Entity storage service.
+   *
+   * @var \Drupal\Core\Entity\EntityStorageInterface
```

Comment:
Additional comment: The data used by the ResourceMapper is stored in ResourceMapping entities.